creator/Documents/CommandBlocks.md

```diff
@@ -25,127 +25,116 @@ In this guide, we will set up a command block chain to look for a certain type o
-    - **Always Active** will set the command block to always activate with or without redstone.
-- **Execute on First Tick** is a boolean (True/False statement) that is used for command blocks set to repeat. If set **True**, the command will activate as soon as it is activated. If set **False**, the command will activate on a delay.
-- **Delay in Ticks** is a value that defines how long a command waits to execute after the command block has been activated.
+    - **Needs Redstone** will set the command block to only activate when it has redstone power.
+    - **Always Active** will set the command block to always activate with or without redstone. For impulse command blocks, this causes them to only run once, so it isn't very useful. For chain command blocks, this is enabled by default.
+- **Execute on First Tick** is only usable for repeating command blocks with a delay. If enabled, the command will execute as soon as the block is powered, and then will wait for the delay before running again. If disabled, the command will not run the first time until the delay has finished.
+- **Delay in Ticks** is a value that defines how long a command block waits to execute after the command block receives redstone power.
 
 ## Designing a Gameplay Loop
 
-A use case of command blocks is allowing creators to create gameplay loops within Minecraft to enrich the player's experience. A simple gameplay loop would be rewarding a player for completing an action. If the gameplay loop was to give a player an emerald for placing a block of wool in a certain location, you can break down each step needed:
-
-1. Check to see if a wool block exists at a set location.
-1. Once a player places a block in the desired location, check to see if the player has already received the reward by checking their score.
-1. If the player hasn't receive the reward, give the player an emerald.
-1. If the player received a emerald, then give them a point.
-
-### Scoreboard
+A use case of command blocks is allowing creators to create gameplay loops within Minecraft. A simple gameplay loop would be rewarding a player for completing an action. If the gameplay loop was to give a player an emerald for placing a diamond block in a certain location, you can break down each step needed:
 
-The Scoreboard is a gameplay system that exists within Minecraft that can keep track of players, objectives, and score values. The scoreboard enables a wide use of options to create unique gameplay styles. In this tutorial, you will have a single objective and value in order to track when a player has completed the task.
-
-1. In Minecraft, open up the chat dialogue box (T or Enter on Windows 10 OS)
-1. Enter the following command: `/scoreboard objectives add wool_placed dummy`.
-    1. The command is adding a brand new objective called Wool_Placed. Dummy fills the argument since scores can only be adjusted via commands.
-1. Enter the following command: `/scoreboard players set @p wool_placed 0`.
```

Comment:
I removed the entire section on scoreboards because this tutorial does not require them, it just makes the concepts more confusing as the guide glosses over details. `/tag` is way easier to understand at a glance.